designer/server/src/routes/forms/create.js

```diff
@@ -294,12 +283,37 @@ export default [
   })
 ]
 
+/**
+ * @param {RequestWithPayload} request
+ * @param {ResponseToolkit} h
+ */
+function redirectToTitleWithErrors(request, h) {
+  const { yar, payload } = request
+
+  yar.flash('validationFailure', {
+    formErrors: create.titleFormErrors,
+    formValues: payload
+  })
+
+  // Redirect POST to GET without resubmit on back button
+  return h.redirect('/create/title').code(303).takeover()
+}
+
 /**
  * @typedef {import('@defra/forms-model').FormMetadataInput} FormMetadataInput
- * @typedef {import('joi').ExternalHelpers} ExternalHelpers
  */
 
 /**
  * @template {import('@hapi/hapi').ReqRef} [ReqRef=import('@hapi/hapi').ReqRefDefaults]
  * @typedef {import('@hapi/hapi').ServerRoute<ReqRef>} ServerRoute
  */
+
+/**
+ * @template {import('@hapi/hapi').ReqRef} [ReqRef=import('@hapi/hapi').ReqRefDefaults]
+ * @typedef {import('@hapi/hapi').Request<ReqRef>} Request
+ */
+
+/**
+ * @typedef {import('@hapi/hapi').ResponseToolkit<any>} ResponseToolkit
+ * @typedef {Request<{ Payload: any }>} RequestWithPayload
```

Comment:
@colinrotherham just adding a note here to remind me to speak to you later about this. We were butting our heads against a wall for a while on this, could do with an advice on this part. I'll DM you later.